py-polars/tests/unit/operations/namespaces/temporal/test_datetime.py

```diff
@@ -260,107 +260,6 @@ def test_dt_datetime_date_time_invalid() -> None:
         pl.Series([date(2020, 1, 1)]).dt.time()
 
 
-@pytest.mark.parametrize(
```

Comment:
drive-by: I've moved all these month-start / month-end tests to a separate file py-polars/tests/unit/operations/namespaces/temporal/test_month_start_end.py . it's all just copied ad-verbatim, with just the `test_month_end_ambiguous_start` test added